deploy/template.yaml

```
@@ -739,8 +751,29 @@ Resources:
         ScaleInCooldown: 420
         ScaleOutCooldown: 60
 
+  ECSPredictiveScalingPolicy:
+    DependsOn: ECSAutoScalingTarget
+    Type: AWS::ApplicationAutoScaling::ScalingPolicy
+    Properties:
+      PolicyName: ECSPredictiveScalingPolicy
+      PolicyType: PredictiveScaling
+      ResourceId: !Join
```

```suggestion
      ResourceId: !Sub service/${AddressFrontEcsCluster}/${AddressFrontEcsService.Name}
```
This should work and is more concise